sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBFileManager.scala

```diff
@@ -146,6 +146,9 @@ class RocksDBFileManager(
 
   private def codec = CompressionCodec.createCodec(sparkConf, codecName)
 
+  private var maxVersion: Option[Long] = None
```

Comment:
Can we rename to maxSeenVersion and minSeenVersion to indicates that it is only an approximate? Also change the method name accordingly.